jmclient/jmclient/blockchaininterface.py

```diff
@@ -567,23 +603,19 @@ def _yield_transactions(self, wallet_name):
     def list_transactions(self, num):
         return []
 
-    def rpc(self, method, args):
-        if method == "listaddressgroupings":
-            raise RuntimeError("default sync not supported by bitcoin-rpc-nohistory, use --recoversync")
-        elif method == "listunspent":
-            minconf = 0 if len(args) < 1 else args[0]
-            maxconf = 9999999 if len(args) < 2 else args[1]
-            return [{
-                "address": self._get_addr_from_desc(u["desc"]),
-                "label": self.wallet_name,
-                "height": u["height"],
-                "txid": u["txid"],
-                "vout": u["vout"],
-                "scriptPubKey": u["scriptPubKey"],
-                "amount": u["amount"]
-                } for u in self.scan_result["unspents"]]
-        else:
-            return super().rpc(method, args)
+    def listaddressgroupings(self):
+        raise RuntimeError("default sync not supported by bitcoin-rpc-nohistory, use --recoversync")
+
+    def listunspent(self, minconf=None):
+        return [{
+            "address": self._get_addr_from_desc(u["desc"]),
+            "label": self.wallet_name,
+            "height": u["height"],
+            "txid": u["txid"],
+            "vout": u["vout"],
+            "scriptPubKey": u["scriptPubKey"],
+            "amount": u["amount"]
+        } for u in self.scan_result["unspents"]]
```

Comment:
There seems to be a bug here related to minconf and maxconf, I've asked @chris-belcher about it on IRC. Will update.

Reply:
He confirms those vars are unused (and un-needed) so we can just drop it here (it wasn't a bug, just a leftover and not used in no-history). i.e. drop the un-used minconfs argument for this implementation of listunspent().